frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -473,11 +468,7 @@ export class HtmlDocument extends React.Component {
     try {
       props.appContentSaveNewComment(state.content, state.timelineWysiwyg, state.newComment, this.setState.bind(this), state.config.slug)
     } catch (e) {
-      if (e === EXCEPTION_MENTION_PARSING) {
-        this.sendGlobalFlashMessage(props.t('Error while detecting the mentions'))
-      } else {
-        this.sendGlobalFlashMessage(props.t('Error while saving the comment'))
-      }
+      this.sendGlobalFlashMessage(e.message ? e.message : props.t('Error while saving the comment'))
```

Comment:
```suggestion
      this.sendGlobalFlashMessage(e.message || props.t('Error while saving the comment'))
```

Reply:
I see that you applied this suggestion elsewhere and resolved this one without applying it. Pick the one you prefer but I suggest picking the same everywhere :-)

Reply:
Sorry, I agree with you, the resolve it's a mistake.